lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1120,6 +1180,81 @@ defmodule Phoenix.LiveViewTest do
     end
   end
 
-  defp proxy_pid(%{proxy: {_ref, _topic, pid}}), do: pid
+  @doc """
+  Returns the Pid of the proxy between between the test and LiveView.
+  """
+  def proxy_pid(%{proxy: {_ref, _topic, pid}}), do: pid
```

Comment:
I would rather duplicate this inside `upload_live_test` and pass the `pid` to the LiveView under test than make this function public. We are only accessing it to test internal behaviour, I don't believe it is generally useful.